Cargo.toml

```diff
@@ -133,7 +133,8 @@ api_identity = { path = "api_identity" }
 approx = "0.5.1"
 assert_matches = "1.5.0"
 assert_cmd = "2.0.12"
-async-bb8-diesel = { git = "https://github.com/oxidecomputer/async-bb8-diesel", rev = "be3d9bce50051d8c0e0c06078e8066cc27db3001" }
+# TODO: Update with a revision before merging
```

Comment:
Note to self: 

- Merge https://github.com/oxidecomputer/async-bb8-diesel/pull/52
- Then update this rev

Reply:
Done